### ISSUE TITLE
[WebHosting] - GIT Integration on web hosting

### ISSUE THREAD

**Issue by guitdev** [ovh/hosting-domain-names-roadmap#27]
Deploy a website, from a GIT repo, on web hosting offers

**Comment by purplizer** [comment]
Hi @GuiF008  It would come in handy if we could set a git deploy location to a different location than the website root folder. For example, in Laravel applications, the website root folder should be the /public/ folder, but the git deploy should be handled in the parent folder.

**Comment by GuiF008** [comment]
Hello @purplizer 

That's possible. 
You configure and deploy to the folder of your choice. 

To do this, you have to create your multisite first, with the target folder of your choice (for example, the /public/ folder that you mentioned), then activate your GIT deployment on that folder.

Does that work for you?
Please feel free to contact me again if necessary.

Br 
@GuiF008

**Comment by purplizer** [comment]
Hi @GuiF008 

No, that does not work. I will give you a concrete example:
- I have the domain test.com
- It is a Laravel-application, so the web root (the target folder in multisite) is not the same as the target folder for the git-deployment, which should happen in the folder above the target folder.
- So for the same domain in the multisite, the git deployment folder is /test.com whereas the web root is /test.com/public
- And you can't set that up in the same multisite-domain.
- So to circumvent this, I had to make an extra subdomain in the multisite called git.test.com which points to the folder /test.com, and the actual domain points to the folder /test.com/public.

So it would actually be interesting if for the same multisitedomain, you could setup the web root to a different location than the git deployment folder, which can't be done at the moment.

**Comment by GuiF008** [comment]
Hi, @purplizer 

I'll check this and let you know soon.

Br 
@GuiF008

**Comment by purplizer** [comment]
Hi @GuiF008 
Is there an update regarding this question? Has this been reviewed, or discarded? Thank you for replying.

**Comment by GuiF008** [comment]
hi @purplizer 

I'm sorry, I haven't received a response to your request yet.
I will contact you and keep you informed as soon as possible.

**Comment by purplizer** [comment]
> hi [@purplizer](https://github.com/purplizer)
> 
> I'm sorry, I haven't received a response to your request yet. I will contact you and keep you informed as soon as possible.

Thank you for your reply. In the meantime I can make do with htaccess files, but it would be really handy if things could work like in Plesk for example.

**Comment by GuiF008** [comment]
Indeed, the configuration via .htaccess meets the need today, and a product-side improvement is being studied to simplify the experience in the long term.
